searchcore/src/vespa/searchcore/proton/matching/match_tools.h

```diff
@@ -138,6 +138,9 @@ class MatchToolsFactory
 
     const Query & query() const { return _query; }
     const RequestContext & getRequestContext() const { return _requestContext; }
+
+    using StringStringMap = std::unordered_map<vespalib::string, vespalib::string, vespalib::hash<vespalib::string>>;
```

Comment:
consider using stllike hash_map for more speed

Reply:
> consider using stllike hash_map for more speed

done.  should we add emplace and other new-fangled stuff to vespalib hash_map maybe?